python-bindings/overview_article/cffi_test.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# IDE might complain with "no module found" here, even when it exists
```

Comment:
This is because it doesn't exist when you start out. :)  CFFI creates the module when you run the build.

Reply:
Yeah, but my IDE, PyCharm, didn't want to acknowledge it, even after making it. I actually lost a lot of time figuring out what the problem is, since I didn't even bother to try and run it, lol

Reply:
that's odd.  I wonder what pycharm is doing, but not enough to actually go find out, you know. :)